lib/opbeat/rack.rb

```diff
@@ -28,16 +28,14 @@ def call(env)
       rescue Error => e
         raise # Don't capture Opbeat errors
       rescue Exception => e
-        evt = Event.from_rack_exception(e, env)
-        Opbeat.send(evt)
+        Opbeat.capture_rack_exception(e, env)
```

Comment:
Sorry If you got confused about removing the `expect(Opbeat).to receive(:send).with(@event)` lines from tests. I put them back since the original issue that was getting the specs to fail was from elsewhere.

The async tests were letting the configuration have `async` callback, and since the middleware now supports `async` behaviour, the specs needed change. I fixed the `async` specs to cleanup after finish, so now that changes are not needed anymore.